packages/eslint-plugin/src/rules/member-ordering.ts

```diff
@@ -99,6 +103,10 @@ const objectConfig = (memberTypes: MemberType[]): JSONSchema.JSONSchema4 => ({
         'natural-case-insensitive',
       ],
     },
+    required: {
```

Comment:
my preference would be to invert the name (and the behaviour too I guess?).
in my experience you think about members as being a "member" or an "optional member", not a "required member" or an "optional member".

🤷 not going to block on it, just adding my 2c

Reply:
How about `optionality?: 'optional-first' | 'required-first'`? Just spitballing...

Reply:
@asdf93074 @bradzacher I'm leaning a bit harder towards `optionality`. Let me know if you'd prefer something else - otherwise I'll work on Monday morning before the release.

Reply:
`firstMembers?: 'required' | 'optional'`

In seriousness though, I'm okay with the above suggestion. Although, `optionality` and `required` do not communicate the full purpose of this configuration (**order** of members based on **optionality**) alone. Maybe `optionalityOrder` or `memberOptionalityOrder` would complete it?

Reply:
I like `optionalityOrder` a lot!

Reply:
+1 to your reasoning, I like `optionalityOrder` a lot! Will go with it now.

I also forgot to check that `member-ordering.md` be updated to reflect the new option - so I'll do that now too. Thanks!

Reply:
> I also forgot to check that member-ordering.md be updated to reflect the new option - so I'll do that now too.

Ah, I was waiting for the core stuff to be approved before I added that in, but thank you for doing it.